app/Http/Controllers/TestController.php

```diff
@@ -0,0 +1,11 @@
+<?php
+namespace App\Http\Controllers;
+
+class TestController extends Controller{
+	public function test(){
+		?:?:
+		return 1;
+		return 2;
+		return 3;
+	}
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->